mgmt/rest/server.go

```diff
@@ -120,27 +121,32 @@ type managesConfig interface {
 }
 
 type Server struct {
-	mm      managesMetrics
-	mt      managesTasks
-	tr      managesTribe
-	mc      managesConfig
-	n       *negroni.Negroni
-	r       *httprouter.Router
-	tls     *tls
-	auth    bool
-	authpwd string
-	addr    net.Addr
-	err     chan error
+	mm         managesMetrics
+	mt         managesTasks
+	tr         managesTribe
+	mc         managesConfig
+	n          *negroni.Negroni
+	r          *httprouter.Router
+	tls        *tls
+	auth       bool
+	authpwd    string
+	addrString string
+	addr       net.Addr
+	wg         sync.WaitGroup
+	killChan   chan struct{}
+	err        chan error
 }
 
+// TODO delete this line
```

Comment:
Oops, had done that and reverted some code so thought it was still fixed. Thanks.